src/index.scss

```diff
@@ -1,56 +1,56 @@
-@import "./ActionRow/ActionRow";
-@import "./Alert/Alert";
-@import "./Annotation/Annotation";
-@import "./Avatar/Avatar";
-@import "./AvatarButton/AvatarButton";
-@import "./Badge/Badge";
-@import "./Breadcrumb/Breadcrumb";
-@import "./Button/Button";
-@import "./Card/Card";
-@import "./Carousel/Carousel";
-@import "./Collapsible/Collapsible";
-@import "./CloseButton/CloseButton";
-@import "./Chip/Chip";
-@import "./Code/Code";
-@import "./Dropdown/Dropdown";
-@import "./Fieldset/Fieldset";
-@import "./Figure/Figure";
-@import "./Form/Form";
-@import "./Hyperlink/Hyperlink";
-@import "./Icon/Icon";
-@import "./Image/Image";
-@import "./Media/Media";
-@import "./Menu/Menu";
-@import "./Modal/Modal";
-@import "./Nav/Nav";
-@import "./Navbar/Navbar";
-@import "./PageBanner/PageBanner";
-@import "./Pagination/Pagination";
-@import "./Popover/Popover";
-@import "./ProgressBar/ProgressBar";
-@import "./OverflowScroll/OverflowScroll";
-@import "./SearchField/SearchField";
-@import "./Scrollable/Scrollable";
-@import "./Sheet/Sheet";
-@import "./Skeleton/Skeleton";
-@import "./Spinner/Spinner";
-@import "./Stepper/Stepper";
-@import "./StatefulButton/StatefulButton";
-@import "./Table/Table";
-@import "./Tooltip/Tooltip";
-@import "./DataTable/DataTable";
-@import "./TransitionReplace/TransitionReplace";
-@import "./ValidationMessage/ValidationMessage";
-@import "./Tabs/deprecated/Tabs";
```

Comment:
I believe this line is causing the build to break (e.g., prevents the Netlify deploy preview from generating). 

```shell
5:28:52 AM:   formatted: Error: Can't find stylesheet to import.\n +
5:28:52 AM:     '   ╷\n' +
5:28:52 AM:     '44 │ @import ./Tabs/deprecated;\n' +
```